test/functional/interface_usdt_net.py

```diff
@@ -78,12 +113,130 @@
     outbound_messages.perf_submit(ctx, &msg, sizeof(msg));
     return 0;
 };
+
+BPF_PERF_OUTPUT(inbound_connections);
+int trace_inbound_connection(struct pt_regs *ctx) {
+    struct NewConnection inbound = {};
+    void* conn_type_pointer;
+    void* address_pointer;
+    bpf_usdt_readarg(1, ctx, &inbound.conn.id);
+    bpf_usdt_readarg(2, ctx, &address_pointer);
+    bpf_usdt_readarg(3, ctx, &conn_type_pointer);
+    bpf_usdt_readarg(4, ctx, &inbound.conn.network);
+    bpf_usdt_readarg(5, ctx, &inbound.existing);
+    bpf_probe_read_user_str(&inbound.conn.addr, sizeof(inbound.conn.addr), address_pointer);
+    bpf_probe_read_user_str(&inbound.conn.type, sizeof(inbound.conn.type), conn_type_pointer);
+    inbound_connections.perf_submit(ctx, &inbound, sizeof(inbound));
+    return 0;
+};
+
+BPF_PERF_OUTPUT(outbound_connections);
+int trace_outbound_connection(struct pt_regs *ctx) {
+    struct NewConnection outbound = {};
+    void* conn_type_pointer;
+    void* address_pointer;
+    bpf_usdt_readarg(1, ctx, &outbound.conn.id);
+    bpf_usdt_readarg(2, ctx, &address_pointer);
+    bpf_usdt_readarg(3, ctx, &conn_type_pointer);
+    bpf_usdt_readarg(4, ctx, &outbound.conn.network);
+    bpf_usdt_readarg(5, ctx, &outbound.existing);
+    bpf_probe_read_user_str(&outbound.conn.addr, sizeof(outbound.conn.addr), address_pointer);
+    bpf_probe_read_user_str(&outbound.conn.type, sizeof(outbound.conn.type), conn_type_pointer);
+    outbound_connections.perf_submit(ctx, &outbound, sizeof(outbound));
+    return 0;
+};
+
+BPF_PERF_OUTPUT(evicted_inbound_connections);
+int trace_evicted_inbound_connection(struct pt_regs *ctx) {
+    struct ClosedConnection evicted = {};
+    void* conn_type_pointer;
+    void* address_pointer;
```

Comment:
nit: this and other occurences
```suggestion
    void *conn_type_pointer = NULL, *address_pointer = NULL;
```